### ISSUE TITLE
Add "refresh" button on "Saved Database Cards" page

### ISSUE THREAD

**Issue by DevGlogicM** [AlanOC91/YGOPRODeck#93]
"Saved Database cards" page should have a button that refreshes the page "without the need of refreshing the entire page in the tab of the browser".

URL of the "Saved Database cards": https://db.ygoprodeck.com/saved/

Thanks.

**Comment by AlanOC91** [comment]
Hi there!

Can you expand on this? Why is a refresh button needed?

The code uses Ajax do when an item is removed it updates immediately (prices and removes cards etc) without needing to refresh the page.

Can you clarify the issue?

Thank you!

**Comment by DevGlogicM** [comment]
Thank you for your interest.

I explain:

By using [this code I found on Stack Overflow](https://stackoverflow.com/a/17748203/4092887), I can get the cards I have in my localStorage.

These are the results:

`["Off","[85489096,68833958,56995655,80863132,54447022,56647086,25262697,16226786,52702748,82112775,69035382,48206762,8955148,72566043,38844957,81756897,69123138,82099401]","ON","PGD"]`

In this test, I have both pages open in tabs on the same window browser - "Google Chrome - Desktop".

Those pages are:

- "Saved Cards" page.

- "[Genex Worker](https://db.ygoprodeck.com/card/?search=Genex%20Worker&_ga=2.171088480.1773843695.1567871098-1430729754.1472409236)" - selected from the main site - it wa a random card appeared.

Now I'm adding the card called "[Genex Worker](https://db.ygoprodeck.com/card/?search=Genex%20Worker&_ga=2.171088480.1773843695.1567871098-1430729754.1472409236)" on the "Saved Cards" - by pressing the "Save Card" button.

I closed the popup window that told me the card was stored in the "Saved Cards" page  - without reloading nor redirecting the page -.

If i check the already openend "Saved cards" page - "without reloading". the previous added card is not shown - it only does _after_ I reload the page.

Hence I consider that "Saved Database cards" page should have a button that refreshes the page "without the need of refreshing the entire page in the tab of the browser".

Thanks :)
